lib/engine/game/g_1822/step/tracker.rb

```diff
@@ -25,6 +25,23 @@ def check_track_restrictions!(entity, old_tile, new_tile)
           super
         end
 
+        def get_tile_lay(entity)
```

Comment:
Might be best modifying the base code to allow a get tile lay without the entity.company? stuff

Reply:
Yes good call.